tests/testthat/test-prediction-column-names.R

```diff
@@ -0,0 +1,52 @@
+suppressPackageStartupMessages(library(tidymodels))
```

Comment:
`test-prediction-column-names.R` is the best name that I could come up with since: it focuses on censored and workflows and we will probably have similar functions to test here in the future.